src/commands/games/mcquery.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable no-control-regex */
+const { CommandStructures, SwitchbladeEmbed, Constants } = require('../../')
+const { Command, CommandParameters, StringParameter } = CommandStructures
+
+module.exports = class MCQuery extends Command {
+  constructor (client) {
+    super(client)
+    this.name = 'mcquery'
+    this.category = 'games'
+    this.aliases = ['minecraftquery']
+    this.parameters = new CommandParameters(this,
+      new StringParameter({ missingError: 'commands:mcquery.noIP' })
+    )
+  }
+  async run ({ t, author, channel }, server) {
+    let address = server.split(':')
+    let port = 25565
+    if (address.length === 2) port = address[1]
+
+    let info = await this.getInfo(address[0], port)
+    let embed = new SwitchbladeEmbed(author)
+    channel.startTyping()
+    if (info.online) {
+      embed.setTitle(`${address[0]}:${port}`)
```

Comment:
Also, maybe don't display the port if it's `25566`?

Reply:
its 25565